ginga/misc/plugins/TVMask.py

```diff
@@ -0,0 +1,447 @@
+"""Non-interactive masks display local plugin for Ginga."""
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+from ginga.util.six import itervalues
+
+# STDLIB
+import re
+import os
+
+# THIRD-PARTY
+import numpy as np
+from astropy.io import fits
```

Comment:
It's no problem to have a plugin that depends on a package, especially `astropy`, which is for all intents and purposes a requirement for the reference viewer.  

In general it *is* nice if we can use the loading abstractions because it makes it easier 1) to test when there is a possible problem with one package (you can try the other), 2) to support new formats more easily with the same plugins.